README.md

```diff
@@ -19,6 +19,31 @@ terraform apply -var-file=<workspace>.tfvars -target=aws_iam_role_policy_attachm
 At this point the `ProvisionNetworking` policy is attached to the
 `ProvisionAccount` role and you can run a full `terraform apply`.
 
+## Pre-requisites ##
+
+- [Terraform](https://www.terraform.io/) installed on your system.
+- An accessible AWS S3 bucket to store Terraform state
+  (specified [here](backend.tf)).
```

Comment:
I'd recommend that instead of using `here` as hyperlink text, to instead use the name of the resource (in this case, `backend.tf`) as it helps with accessibility.

Reply:
Thanks- updated in https://github.com/cisagov/cool-sharedservices-networking/pull/23/commits/dd20effaf477fd82541da2b1b4f3ed22ed51d316 (similar changes made in the other PRs mentioned in https://github.com/cisagov/cool-system/issues/58).